src/Categories/Category/Construction/Graphs.agda

```diff
@@ -529,12 +529,12 @@ CatF-is-Free o = record
     unwind-resp-≈ ε = Equiv.refl
     unwind-resp-≈ (x ◅ eq) = ∘-resp-≈ (unwind-resp-≈ eq) x
 
-  zig′ : (X : Graph o o o) → {A B : Graph.Obj X} → (f : Star (Graph._⇒_ X) A B) →
-    let Y = Free X in [ X ] (unwind Y) (mapGraph (GM X) f) ≈* f
```

Comment:
I have no idea why this originally was a `let ... in` construct. It seemed pointless to me so I refactored it but if I'm missing something let me know so I can change it back.